lab-ron-dunphy/package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "lab-ron-dunphy",
   "version": "1.0.0",
-  "description": "",
+  "description": "lab 01",
   "main": "index.js",
   "directories": {
-    "test": "test"
+    "test": "test",
+    "test": "gulp",
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha; gulp",
```

Comment:
Looks like you still need to push the correction on this line. These scripts need to be their own commands.